src/api/entities/Asset/Fungible/CorporateActions/Ballots.ts

```diff
@@ -49,47 +51,23 @@ export class Ballots extends Namespace<FungibleAsset> {
    * @throws if the provided Corporate Action does not exist
    */
   public async getOne(args: { id: BigNumber }): Promise<CorporateBallot> {
```

Comment:
```suggestion
  public async getOne(args: { id: BigNumber }): Promise<CorporateBallotWithDetails> {
```
Similar to dividend distribution, we should create a separate interface here `CorporateBallotWithDetails` 
```
export interface BallotWithDetails {
  ballot: CorporateBallot;
  details: CorporateBallotDetails;
}
```
Ballot details should not be clubbed within entity info